models/models.js

```diff
@@ -98,4 +98,41 @@ exports.updateArticleVotes = (articleId, incVotes) =>{
 
     return db.query(`UPDATE articles SET votes = votes + $2 WHERE article_id = $1 RETURNING *;`, [articleId, incVotes])
     .then((dbResponse)=>{return dbResponse.rows[0]})
+}
+
+
+
+exports.checkCommentExists = (commentId) =>{
+   console.log(Number(commentId))
```

Comment:
Please remove all console logs from code you want reviewed. The handler below is not needed, PSQL will reject for you if the id is not the correct type.